datasets/plugins/register_plugins.py

```diff
@@ -0,0 +1,18 @@
+from datasets.dataset_plugin import DatasetPlugin
+from datasets.plugins import MetaflowExecutor
+
+
+def register():
+    from importlib_metadata import entry_points
+
+    # Register plugins
+    for entry in entry_points(group="datasets.plugins"):
```

Comment:
That's super cool, learning new features of `python` reviewing this PR!

Reply:
Saved me a ton of work!